config.js

```diff
@@ -23,16 +23,33 @@ const config = {
     logoLink: 'https://www.prisma.io',
     title: 'Prisma',
     links: [
-      { name: 'Docs', link: '/' },
-      { name: 'Quickstart', link: '/getting-started/quickstart' },
-      {
-        name: 'Reference',
-        link: '/reference/tools-and-interfaces/prisma-client/',
-      },
+      { name: 'Documentation', link: '/' },
+      // {
+      //   name: 'Reference',
+      //   link: '/reference/tools-and-interfaces/prisma-client/',
+      // },
       { name: 'Blog', link: 'https://www.prisma.io/blog/' },
       { name: 'Community', link: 'https://www.prisma.io/community/' },
-      { name: 'FAQ', link: '/more/faq' },
-      { name: 'Prisma 1', link: 'https://www.prisma.io/docs/1.34' },
+      // { name: 'FAQ', link: '/more/faq' },
+      // { name: 'Prisma 1', link: 'https://www.prisma.io/docs/1.34' },
+      { name: 'Quickstart', link: '/getting-started/quickstart', type: 'button' }
```

Comment:
should made to
```suggestion
      { name: 'Quickstart', link: '/getting-started/quickstart-typescript', type: 'button' }
```
as doc sidebar 'Quickstart' links to quickstart-typescript